packages/dds/sequence/src/mapKernel.ts

```diff
@@ -499,8 +507,14 @@ export class MapKernel implements IValueTypeCreator {
     public trySubmitMessage(op: any, localOpMetadata: unknown): boolean {
         const type: string = op.type;
         if (this.messageHandlers.has(type)) {
+            const mapLocalMetadata: Partial<IMapMessageLocalMetadata> = localOpMetadata;
+            // we don't know how to rebase these operations, so if any other op has come in
+            // we will fail.
+            if(this.lastProcessedSeq !== mapLocalMetadata?.lastProcessedSeq) {
+                throw new Error("SharedInterval does not support reconnect in presence of external changes");
```

Comment:
i'd forgotten about the read-write transition which happens frequently and depends on reconnect. so i added some simple logic that tracks the last op seq, and allows resend if there are no other changes targeting this dds, basically this allows reconnect as long as no one else is doing anything